tests/Dockerfile

```diff
@@ -0,0 +1,15 @@
+ARG PYTHON_VERSION=3.10
+FROM python:${PYTHON_VERSION}-alpine3.21
+
+WORKDIR /tests
+
+ARG TARGETPLATFORM
+
+RUN apk add --no-cache python3 py-pip curl uv
+
+RUN python3 -m venv /venv
+
+ADD requirements.txt .
```

Comment:
Here better change ADD -> COPY
https://www.docker.com/blog/docker-best-practices-understanding-the-differences-between-add-and-copy-instructions-in-dockerfiles/